starlette/requests.py

```diff
@@ -82,15 +87,15 @@ def query_params(self) -> QueryParams:
             elif message["type"] == "http.disconnect":
                 raise ClientDisconnect()
 
-    async def body(self):
+    async def body(self) -> bytes:
         if not hasattr(self, "_body"):
             body = b""
             async for chunk in self.stream():
                 body += chunk
             self._body = body
         return self._body
 
-    async def json(self):
+    async def json(self) -> JSONType:
```

Comment:
I'm happy with just `typing.Any` here, since we can't properly type json anyways.

Reply:
perfect!